src/modules/nftCollections/collections-getter.service.ts

```diff
@@ -309,21 +307,20 @@ export class CollectionsGetterService {
     return [groupedCollections, groupedCollections.length];
   }
 
-  private async getMappedCollections(page: number, size: number): Promise<Collection[]> {
-    const collections = await this.apiService.getCollections(new CollectionQuery().addPageSize(page, size).build());
-    const promisesCollections = collections?.map((collection): Promise<Collection> => this.mapCollection(collection));
+  private async getMappedCollections(items): Promise<Collection[]> {
+    const promisesCollections = items?.map((collection): Promise<Collection> => this.mapCollection(collection));
     return await Promise.all(promisesCollections);
   }
 
-  private async mapCollection(collection: CollectionApi): Promise<Collection> {
+  private async mapCollection(collection: any): Promise<Collection> {
```

Comment:
Is there a specific entity for collection instead of `any` ?

Reply:
done